src/sentry/incidents/logic.py

```diff
@@ -39,6 +38,7 @@
 from sentry.incidents import tasks
 from sentry.utils.committers import get_event_file_committers
 from sentry.utils.snuba import (
+    _snuba_pool,
```

Comment:
Importing things with `_` makes me 🤔 Should we make this a non-private symbol or add a better API?

Reply:
I'd love to get this merged first to unbreak the tests and prod and then follow up with improvements if that fits your bill.

Just pushed a merge from master, fixing the `black` conflicts.

Reply:
@markstory I don't love it either, but I think we need to move the snuba stuff around a bit in general. The new `snuba` app I added in https://github.com/getsentry/sentry/pull/14383 could be a decent place.